docs/userGuide/syntax/boxes.mbdf

```diff
@@ -101,6 +155,9 @@
 <box type="info" icon=":rocket:">
     info, with rocket icon
 </box>
+<box type="definition" light>
+    light definition
+</box>
```

Comment:
Since this is already displayed above, this code doesn't seem to be necessary anymore?

Reply:
Ah sorry forgot to push latest commit. will fix